docs/reference/create.md

```diff
@@ -126,7 +126,7 @@ Labels:
 The supported flags are as follows:
 
 - `--engine-insecure-registry`: Specify [insecure registries](https://docs.docker.com/reference/commandline/cli/#insecure-registries) to allow with the created engine
-- `--engine-registry-mirror`: Specify <a href="https://github.com/docker/docker/blob/master/docs/sources/articles/registry_mirror.md">registry mirrors</a> to use
+- `--engine-registry-mirror`: Specify <a href="https://github.com/docker/docker/blob/master/docs/articles/registry_mirror.md">registry mirrors</a> to use
```

Comment:
This doc is empty / confusing.
We should rather link to https://github.com/docker/distribution/blob/master/docs/mirror.md